internal/pkg/describe/errors.go

```diff
@@ -15,3 +20,13 @@ type ErrManifestNotFoundInTemplate struct {
 func (err *ErrManifestNotFoundInTemplate) Error() string {
 	return fmt.Sprintf("manifest metadata not found in template of stack %s-%s-%s", err.app, err.env, err.name)
 }
+
+type errLbWebSvcsOnCFWithoutAlias struct {
+	services []string
+}
+
+// Error implements the error interface.
+func (err *errLbWebSvcsOnCFWithoutAlias) Error() string {
+	return fmt.Sprintf("%v %v must have http.alias specified when CloudFront is enabled", english.PluralWord(len(err.services), "service", "services"),
```

Comment:
```suggestion
	return fmt.Sprintf("%v %v must have %q specified when CloudFront is enabled", english.PluralWord(len(err.services), "service", "services"),
```
maybe it'd be better to quote "http.alias"